internal/config/config.go

```diff
@@ -112,11 +113,16 @@ func LoadFirmwareManifest(ctx context.Context, manifestURL string) (map[string][
 	for _, m := range models {
 		for component, firmwareRecords := range m.Components {
 			for _, fw := range firmwareRecords {
+				cModels := []string{strings.ToLower(m.Model)}
```

Comment:
Would this mean all `Model` attributes in the `ComponentFirmwareVersion` will unconditionally include the server model as well? Can we have a test for this method, that would help clarify the input data and output records.

Reply:
Yes, that's how it is atm. Most of the firmware files are organized per server model in `modeldata.json`, this change will add to the model list any specific component model defined. I added a test case that shows how the data will look like in [fe64406](https://github.com/metal-toolbox/firmware-syncer/pull/84/commits/fe64406da735642a374b8c91beae487b0406624a)